src/mem.rs

```diff
@@ -223,6 +262,49 @@ impl Compress {
             }
         }
     }
+}
+
+impl Builder<Decompress> {
+    /// Create a `Decompress` object based on the configured values.
+    pub fn finish(&mut self) -> Decompress {
+        unsafe {
+            let mut state = ffi::StreamWrapper::default();
+            let ret = ffi::mz_inflateInit2(
+                &mut *state,
+                if self.zlib_header {
+                    self.window_bits
+                } else {
+                    -self.window_bits
+                },
+            );
+            debug_assert_eq!(ret, 0);
```

Comment:
Woudln't it be best to have the assert in release too? Otherwise there will be issues if inflateInit fails.

Reply:
Which it might since [miniz](https://github.com/alexcrichton/flate2-rs/blob/master/miniz-sys/miniz.c#L195) and miniz_oxide only support one window size

Reply:
Thank you! I applied your suggestion and also changed `Builder::window_bits` to only be available for the `zlib` feature.